server/views/pages/product.njk

```diff
@@ -23,7 +23,7 @@
     </tr>
     <tr>
       <th>ID</th>
-      <td>{{ product.id }}</p></td>
+      <td>{{ product.id }}</td>
```

Comment:
This led to weirdly formatted data:
<kbd>
<img width="204" alt="Screenshot 2024-08-29 at 12 15 51" src="https://github.com/user-attachments/assets/462fff40-842b-441e-981f-fa93f8c84997">
</kbd>